docs/how-to/command-line-quick-start.md

```diff
@@ -1,178 +1,243 @@
-> to get started, enter:
->
->   ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme
-```
+1. Open a terminal window.
 
-If you are running on a server in a data center, you should initialize IPFS with the `server` profile. Doing so will prevent IPFS from creating a lot of data center-internal traffic trying to discover local nodes:
+1. Initialize the repository with the `ipfs init` command
 
-```bash
-ipfs init --profile server
-```
+    ```bash
+    ipfs init
+    ```
 
-There are a whole host of other configuration options you may want to set — check [the full reference](https://github.com/ipfs/kubo/blob/master/docs/config.md) for more.
+   Output similar to the following displays:
 
-The hash after `peer identity:` is your node’s ID and will be different from the one shown in the above output. Other nodes on the network use it to find and connect to you. You can run `ipfs id` at any time to get it again if you need it.
+    ```bash
+    > initializing ipfs node at /Users/jbenet/.ipfs
+    > generating 2048-bit RSA keypair...done
+    > peer identity: Qmcpo2iLBikrdf1d6QU6vXuNb6P7hwrbNPW9kLAH8eG67z
+    > to get started, enter:
+    >
+    >   ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme
+    ```
+    
+    :::tip
+    The hash next to `peer identity` is your node’s ID and will be different from the one shown in the above output. Other nodes on the network use `peer identity` to find and connect to you. 
+    
+    Run `ipfs id` to display the `peer identity` if you need it.
+    :::
 
-Now, try running the command suggested to you in the output of `ipfs init`. The one that looks like `ipfs cat /ipfs/<HASH>/readme`.
 
-You should see something like this:
 
-```
-Hello and Welcome to IPFS!
-
-██╗██████╗ ███████╗███████╗
-██║██╔══██╗██╔════╝██╔════╝
-██║██████╔╝█████╗  ███████╗
-██║██╔═══╝ ██╔══╝  ╚════██║
-██║██║     ██║     ███████║
-╚═╝╚═╝     ╚═╝     ╚══════╝
-
-If you see this, you have successfully installed
-IPFS and are now interfacing with the ipfs merkledag!
-
- -------------------------------------------------------
-| Warning:                                              |
-|   This is alpha software. use at your own discretion! |
-|   Much is missing or lacking polish. There are bugs.  |
-|   Not yet secure. Read the security notes for more.   |
- -------------------------------------------------------
-
-Check out some of the other files in this directory:
-
-  ./about
-  ./help
-  ./quick-start     <-- usage examples
-  ./readme          <-- this file
-  ./security-notes
-```
+2. Now, try running the command suggested to you in the output of `ipfs init` (i.e. `ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme`):
 
-You can explore other objects in the repository. In particular, the `quick-start` directory which shows example commands to try:
+   ```bash
+   ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme
```

Comment:
Probably time to:
- Update this file and remove the alpha software warning
- use a CidV1

Reply:
@ElPaisano task
- [ ] create PR to update https://github.com/ipfs/kubo/blob/master/assets/init-doc/readme


@2color RE use a CidV1:
Are you saying we should update this tutorial to use CIDv1 instead of CIDv0? Make sense to me. 

Trying to dogfood our own docs here, I see https://docs.ipfs.tech/how-to/best-practices-for-ipfs-builders/#use-cidv1-for-future-proof-addressing-and-case-insenstive-contexts -> should we just add a step in to this telling the user to run `ipfs add --cid-version 1` (this is probably a good opportunity for me to get more hands on with IPFS)

Also, seems like that comment could apply to all docs in the tutorial, so seems like we should create an issue to update tutorials to use CIDv1?